pkg/stream/operators.go

```diff
@@ -216,7 +216,12 @@ func Debounce[T any](src Observable[T], duration time.Duration) Observable[T] {
 						complete(err)
 						return
 
-					case item := <-items:
+					case item, ok := <-items:
```

Comment:
isn't this an infinite loop when items is closed? a read from a closed channel can always be chosen by `select`, AFAIK

Reply:
Damn, you are right. `item` should be set to nil in that case.

Reply:
Fixed now.